docs/continuous-integration.md

```diff
@@ -0,0 +1,19 @@
+# ONS — Continuous Integration
```

Comment:
I think we should just capture this as part of the readme as this feel like a duplicate. We can reintroduce later where we have a standards doc that link to type hints, code styles etc.